app/Http/Controllers/Controller.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @SWG\Swagger(
- *     basePath="/api",
+ *     basePath="/",
```

Comment:
I changed this because I wanted the `/open`-prefixed services to be featured in the Swagger documentation, not just the `/api`-prefixed services.